### ISSUE TITLE
1.11 breaks extraction from ogg files

### ISSUE THREAD

**Issue by michihenning** [taglib/taglib#770]
I have distilled the attached example from a much larger code base. This is code that is currently deployed and working with 1.9.1.

[taglib-bug.zip](https://github.com/taglib/taglib/files/582253/taglib-bug.zip)

Compile with
c++ --std=c++11 ./bug.cpp -ltag

With 1.9.1, this prints
Found 4020 bytes of image data

When compiling and linking with 1.11.1, the code asserts on line 21 because the returned map is empty.

Can anyone suggest a work-around? This is critical for us.

**Comment by TsudaKageyu** [comment]
Sorry for inconvenience. Now, `XiphComment` handles the pictures separately and parse them automatically. 
Please check our [API documentation](http://taglib.org/api/classTagLib_1_1Ogg_1_1XiphComment.html#adb32cb57707577dd6e4c83f92656ccdf).

**Comment by michihenning** [comment]
Yes, I worked out how to fix it. But why break the behavior of the existing API? The previously working code still compiles but fails at run time. Why am I no longer allowed to look at the Vorbis comments?
